src/components/Slider/Slider.stories.tsx

```diff
@@ -0,0 +1,136 @@
+import { BADGE } from '@geometricpanda/storybook-addon-badges';
+import type { StoryObj, Meta } from '@storybook/react';
+import { userEvent } from '@storybook/testing-library';
+import React, { useState } from 'react';
+
+import { Slider } from './Slider';
+
+export default {
+  title: 'Components/Slider',
+  component: Slider,
+  parameters: {
+    layout: 'centered',
+    badges: [BADGE.BETA],
+  },
+  decorators: [
+    (Story) => (
+      <div className="w-96">
+        <Story />
+      </div>
+    ),
+  ],
+} as Meta<Args>;
+
+type Args = React.ComponentProps<typeof Slider>;
+
+const InteractiveSlider = ({
+  min = 0,
+  max = 100,
+  step = 1,
+  value = 50,
+  ...args
+}: Args) => {
+  const [sliderValue, setSliderValue] = useState(value);
+  return (
+    <Slider
+      max={max}
+      min={min}
+      step={step}
+      {...args}
+      onChange={(e) => setSliderValue(Number(e.target.value))}
+      value={sliderValue}
+    />
+  );
+};
+
+export const Default: StoryObj<Args> = {
+  args: {
+    label: 'Slider Label',
+  },
+  render: (args) => <InteractiveSlider {...args} />,
```

Comment:
nit: it seems like the default export could specify `render` and remove the `render` prop from each of the individual stories

Reply:
Ahh nice, didn't think to put the `render` fn in the default export, done